### PR TITLE
Adds similar language for nav and title issue 7249

### PR DESCRIPTION
**Pull request by MiraMarshall**
_Description of what this PR is changing or adding, and why:_
The PR adds similar language for the keyboard focus system nav and page title. 
_Issues fixed by this PR (if any):_
#7249 
## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/_data/sidenav.yml [hunk 1]
```diff
@@ -106,7 +106,7 @@
           children:
             - title: Actions & shortcuts
               permalink: /development/ui/advanced/actions_and_shortcuts
-            - title: Keyboard focus
+            - title: Keyboard focus system
```
Comment (domesticmouse) [inline src/_data/sidenav.yml:109]: Is this title going to be too wide for the sidebar?

**Review by domesticmouse — APPROVED** [review]

File: src/_data/sidenav.yml (hunk 1 above)
Comment (sfshaza2) [reply src/_data/sidenav.yml:109]: It should be ok. It does wrap text.

Comment (domesticmouse) [reply src/_data/sidenav.yml:109]: Glad to see you are totally doing your holiday in style =)

Comment (sfshaza2) [reply src/_data/sidenav.yml:109]: LOL. What can I say? Typical! But, I remember you did something similar not so long ago.  🗡️ 

Honestly, I suspect the baby has been born but he's in no rush to tell us.  ;)

**Pull request merged**